docs/source/dataset_summary.rst

```diff
@@ -31,6 +31,17 @@ Columns definitions:
     - **#Runs** is the number of runs per session. A run is a continuous recording of the EEG data. Often, the different runs of a given session are recorded without removing the EEG cap in between.
     - **PapersWithCode leaderboard** is the link to the dataset on the PapersWithCode leaderboard.
 
+**Datasets overview:**
+
+A visual overview of all datasets can be generated using the function :func:`moabb.datasets.utils.plot_all_datasets`.
+This overview allows to quickly compare the number of subjects, trials, and sessions across different datasets.
+The function will generate a figure like this:
+
+.. figure:: images/datasets_overview.png
+   :alt: Datasets overview
+   :width: 100%
```

Comment:
@bruAristimunha what do you think of adding this paragraph to the dataset summary page?

Reply:
perfect for me, super nice pics btw